projects/harfbuzz/build.sh

```diff
@@ -26,13 +26,17 @@ export CXXFLAGS="$CXXFLAGS -fno-sanitize=function,vptr"
 ./configure
 make clean
 make -j$(nproc) V=1 all
-make -C src V=1 fuzzing
+make CPPFLAGS="-DHB_NO_VISIBILITY" -C src V=1 fuzzing
 
 # Build the fuzzer.
 $CXX $CXXFLAGS -std=c++11 -Isrc \
-    ./test/fuzzing/hb-fuzzer.cc -o $OUT/hb-fuzzer \
+    ./test/fuzzing/hb-shape-fuzzer.cc -o $OUT/hb-shape-fuzzer \
```

Comment:
Would it be possible to move these upstream, and have here only something like "make fuzzers" or whatever build system is used. We propose it to every project as part of the ideal integration (https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md).

Not a blocker for this PR, but would be a great improvement in future.

Reply:
Acknowledged, I'll plan to do this separate from this PR.